apps/condo/domains/billing/components/OnBoarding/SetupAcquiring.tsx

```diff
@@ -0,0 +1,132 @@
+import get from 'lodash/get'
+import { useRouter } from 'next/router'
+import React, { useCallback, useEffect } from 'react'
+
+import { useOrganization } from '@open-condo/next/organization'
+import { Typography } from '@open-condo/ui'
+
+import { AcquiringIntegrationContext as AcquiringContext, AcquiringIntegration } from '@condo/domains/acquiring/utils/clientSchema'
+import { BillingIntegrationOrganizationContext as BillingContext } from '@condo/domains/billing/utils/clientSchema'
+import { Loader } from '@condo/domains/common/components/Loader'
+import { extractOrigin } from '@condo/domains/common/utils/url.utils'
+import { IFrame } from '@condo/domains/miniapp/components/IFrame'
+import { CONTEXT_FINISHED_STATUS, CONTEXT_IN_PROGRESS_STATUS } from '@condo/domains/miniapp/constants'
+
+type SetupAcquiringProps = {
+    onFinish: () => void
+}
+
+export const SetupAcquiring: React.FC<SetupAcquiringProps> = ({ onFinish }) => {
+    const router = useRouter()
+    const { organization } = useOrganization()
+    const orgId = get(organization, 'id', null)
+
+    const createAction = AcquiringContext.useCreate({
+        status: CONTEXT_IN_PROGRESS_STATUS,
+        settings: { dv: 1 },
+        state: { dv: 1 },
+    })
+    const updateAction = AcquiringContext.useUpdate({
+        status: CONTEXT_FINISHED_STATUS,
+    })
+
+    const { obj: billingCtx, loading: billingCtxLoading, error: billingCtxError } = BillingContext.useObject({
+        where: {
+            status: CONTEXT_FINISHED_STATUS,
+            organization: { id: orgId },
+        },
+    })
+
+    // NOTE: On practice there's only 1 acquiring and there's no plans to change it soon
+    const { objs: acquiring, loading: acquiringLoading, error: acquiringError } = AcquiringIntegration.useObjects({
```

Comment:
Maybe it's better to use `.useObject(...)` to be sure there is only one object?

Reply:
Nah, we need to take first one if multiple found...It's safer than to throw a error here